.github/workflows/test.yml

```diff
@@ -0,0 +1,69 @@
+name: TCC GitHub Action
+on:
+  pull_request:
+    branches: [main]
+  push:
+    branches: [main]
+jobs:
+  defaults:
+    runs-on: ubuntu-latest
```

Comment:
I think we can add the macos and windows and make sure it works on all of them

Reply:
done